spec/requests/api/v1/locations_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe "Locations API" do
+  it "can create a location" do
+    post "/api/v1/locations", params: { location: { creator: "John", physical_address: "Cr 70C No. 4-42 OF 203, Medellín", city: "Medellín", state: "Medellín", country: "Colombia", ethereum_address: "0x7da07904ffc79d7a69afdecb7b1eb0d767b37803"}}
+
+    expect(response.status).to eq 201
+    expect(Location.first.creator).to eq "John"
```

Comment:
@DavidKnott I think we should be checking city, state and country too - just to make sure it's all wired up.